package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "Symphony",
-  "productName": "Symphony",
+  "productName": "Symphony - ODP",
   "version": "1.0.1",
-  "description": "Symphony desktop app",
+  "description": "Symphony desktop app (Foundation ODP)",
```

Comment:
Because it's more likely that the typical contributor to or consumer of SymphonyElectron has access to the ODP than the Symphony corporate pod.